adafruit_mcp2515/__init__.py

```diff
@@ -324,33 +324,20 @@ def initialize(self):
 
         self._set_mode(new_mode)
 
-    def send(self, message_obj, wait_sent=True):
+    def send(self, message_obj):
         """Send a message on the bus with the given data and id. If the message could not be sent
          due to a full fifo or a bus error condition, RuntimeError is raised.
 
         Args:
             message (canio.Message): The message to send. Must be a valid `canio.Message`
         """
+
         # TODO: Timeout
         tx_buff = self._get_tx_buffer()  # info = addr.
+        if tx_buff is None:
+            raise RuntimeError("No transmit buffer available to send")
```

Comment:
OK this does implement what the core docs say.  It's not what core does.  sigh.  We can let some actual experience inform us as to what is preferable, and then implement it across the various chips, I don't want it to hold things back at this point.

Reply:
I agree that we should wait and see how things go.